### PR TITLE
contribution: deleted records

### PR DESCRIPTION
**Pull request by rerowep**
* Adds deleted field to json schema et mapping.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: rero_ils/modules/contributions/jsonschemas/contributions/contribution-v0.0.1.json [hunk 1]
```diff
@@ -80,6 +80,10 @@
     "idref": {
       "title": "IdRef authorities",
       "type": "object"
+    },
+    "deleted": {
+      "title": "Deletion date",
```
Comment (iGormilhit) [inline rero_ils/modules/contributions/jsonschemas/contributions/contribution-v0.0.1.json:85]:
```suggestion
    "deletion_date": {
      "title": "Deletion date",
```

File: rero_ils/modules/contributions/mappings/v7/contributions/contribution-v0.0.1.json [hunk 2]
```diff
@@ -359,6 +359,9 @@
       "organisations": {
         "type": "keyword"
       },
+      "deleted": {
```
Comment (iGormilhit) [inline rero_ils/modules/contributions/mappings/v7/contributions/contribution-v0.0.1.json:362]:
```suggestion
      "deletion_date": {
```

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit message corrections:

```
document: identify deleted contribution records 

* Adds a deletion_date field to JSON schema and mapping in order to track when
  a MEF contribution record is deleted.
```

File: rero_ils/modules/contributions/jsonschemas/contributions/contribution-v0.0.1.json (hunk 1 above)
Comment (rerowep) [reply rero_ils/modules/contributions/jsonschemas/contributions/contribution-v0.0.1.json:85]: If I change this I have to change it too in rero-mef. We can open an issue and change it in a new release of rero-mef.

Comment (rerowep) [reply rero_ils/modules/contributions/jsonschemas/contributions/contribution-v0.0.1.json:85]: There are also date fields named `_updated` and `_changed`.

File: rero_ils/modules/contributions/mappings/v7/contributions/contribution-v0.0.1.json (hunk 2 above)
Comment (rerowep) [reply rero_ils/modules/contributions/mappings/v7/contributions/contribution-v0.0.1.json:362]: If I change it, I have to change it in rero-mef too. There other date fields named `_updated` and `_created`.

**Comment by iGormilhit** [comment]
@rerowep And, I don't get why you're opening a PR on the master branch. What are you going to deploy?

**Pull request closed without merging**